### DISCUSSION TITLE
Are Allure2 and TestOps supporting Azure Devops?

### DISCUSSION THREAD

**Discussion by KristofKuli** [allure-framework/allure2#1926] [Questions & Support] [answered]
Hi,

Currently we have Allure1 installed as an extension in our Azure Devops organization, but since there's no support to it anymore, we want to swap to Allure2. Is it possible to install it as an extension? I couldn't find any relevant information nor the new version in the Azure Devops Marketplace.

Thanks,
Kristof

**Comment by cheshi-mantu** [comment]
@KristofKuli 
Hello there!
Allure TestOps suports working with Azure Devops, please refer to the docs page: https://docs.qameta.io/allure-testops/upload/azure/

**Reply by KristofKuli** [reply]
Hi!

Thank you for your reply. So if I get this right, for this version it's not possible to display the report in HTML format, right under the finished run in Azure Devops? We can only deploy to TestOps?
I am pasting the Marketplace page for reference:
https://marketplace.visualstudio.com/items?itemName=Molecula.allure-test-reports

**Reply by KristofKuli** [reply]
Hi,

Any update on this?

**Reply by cheshi-mantu** [reply]
Unfortunately, it does not work in Azure Devops, please refer to this issue: https://github.com/microsoft/azure-pipelines-task-lib/issues/909
We're still trying to come up with a workaround, but no luck at the moment.

**Comment by baev** [comment, marked as answer]
Official Allure Report extension for Azure Pipelines are now available https://marketplace.visualstudio.com/items?itemName=qameta.allure-azure-pipelines

History isn't yet supported, but we'll get there! Please submit feedback to our [discussions](https://github.com/orgs/allure-framework/discussions)
